scripts/oes-data-migration-scripts/migration_v3.12.x_to_v4.0.x.py

```diff
@@ -267,7 +267,7 @@ def update_approval_gate_url(json_data, pl_key, execution_str):
     except Exception as e:
         print("Exception occurred while updating approval gate navigation url : ", e)
         logging.error("Exception occurred while updating approval gate navigation url : ", exc_info=True)
-        raise e
```

Comment:
@Pranav-b-7 now when you are not throwing exception, shouldn't we set " is_error_occurred = True " so the overall migration staus is shown failed ? Otherwise, user might be under impression that migraion is succesful but it might have failed here!

Reply:
@Pranav-b-7 please relook at this change. Raising exception in these methods will break the loop in caller which I think we didn't want so, we ate the exception. With that, we should stick to not raising exception so the loop doesn't break and just set the flag in these three methods to indicate failure. Please cross check my statements before you act (we can connect if needed)!